cfssl_scan.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/cloudflare/cfssl/scan"
+)
+
+var scanUsageText = `cfssl scan -- scan a host for issues
+
+Usage of scan:
+        cfssl scan [-short] [-scanfamily regexp] [-scanner regexp] HOST+
+        cfssl scan -list [-short] [-scanfamily regexp] [-scanner regexp]
+
+Arguments:
+        HOST:    Host(s) to scan (including port)
+Flags:
+`
+var scanFlags = []string{"short", "list", "family", "scanner"}
+var scanFamilyRegexp, scannerRegexp *regexp.Regexp
+
+func regexpLoop(familyFunc func(*scan.Family) error, scannerFunc func(*scan.Scanner) error) (err error) {
```

Comment:
I think its worth documenting these arguments as they are rather complex and it's not immediately clear what's expected of a familyFunc or a scannerFunc.